stable/spinnaker/templates/rbac/halyard-sa.yaml

```diff
@@ -10,4 +10,8 @@ metadata:
   namespace: {{ .Release.Namespace }}
   labels:
 {{ include "spinnaker.standard-labels" . | indent 4 }}
+  {{- if .Values.serviceAccount.serviceAccountAnnotations }}
+  annotations:
+{{ toYaml .Values.serviceAccount.serviceAccountAnnotations | indent 4 }}
```

Comment:
You need to fix the indentation try `nindent 6`

Reply:
I would have thought 4 was correct? an indent of 2 below annotations?